index.html

```diff
@@ -1,59 +1,225 @@
+				<nav class="navlinks">
+					<ul class="links">
+						<li><a href="">Portfolio</a></li>
+						<li><a href="">About</a></li>
+						<li><a href="">Contact</a></li>
+					</ul>
+				</nav>
+				<div class="toggleMenu">
+					<div class="menu"></div>
+				</div>
+			</header>
 
-      <section class="hero-intro">
-        <div class="title">
-          <h1>
-            Hey there. I’m Evans. <br />
-            <span class="inner-h1"> I’m a software developer</span>
-          </h1>
-        </div>
-        <div class="description">
-          <p>
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have a
-            project you need coded, don’t hestiate to contact me.
-          </p>
-        </div>
-        <ul class="socials">
-          <li><a href=""><i class="fa-brands fa-github"></i></a></li>
-          <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
-          <li><a href=""><img src="/assets/Vector.png" alt="" /></a></li>
-          <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
-          <li><a href=""><i class="fa-solid fa-m"></i></a></li>
-        </ul>
+			<section class="hero-intro">
+				<div class="title">
+					<h1>
+						Hey there. I’m Evans. <br />
+						<span class="inner-h1"> I’m a software developer</span>
+					</h1>
+				</div>
+				<div class="description">
+					<p>
+						I can help you build a product , feature or website Look through
+						some of my work and experience! If you like what you see and have a
+						project you need coded, don’t hestiate to contact me.
+					</p>
+				</div>
+				<ul class="socials">
+					<li>
+						<a href=""><i class="fa-brands fa-github"></i></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+					</li>
+					<li>
+						<a href=""><img src="/assets/Vector.png" alt="" /></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-brands fa-twitter"></i></a>
+					</li>
+					<li>
+						<a href=""><i class="fa-solid fa-m"></i></a>
+					</li>
+				</ul>
 
-        <div class="scroll">
-            <span><i class="fa-solid fa-chevron-down"></i></span>
-        </div>
-      </section>
-    </section>
-    <script src="main.js"></script>
-  </body>
+				<div class="scroll">
+					<span><i class="fa-solid fa-chevron-down"></i></span>
+				</div>
+			</section>
+		</section>
+
+		<section id="portfolio">
+			<div class="ttl">
+				<h2>My Recent Works</h2>
+			</div>
+			<div class="cards">
+				<div class="card cd1">
+					<div class="img-hdr"><img src="" alt="" /></div>
+					<div class="langs">
+						<h3>
+							Multi-Post Stories <br />
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
```

Comment:
- [ ] As stated the project requirement Kindly Implement the button interactions follow the button styling: Active, Hover and Pressed (see design).

![Capture](https://user-images.githubusercontent.com/93598189/212038100-cbfbf329-e603-4e98-a910-5619b51b0295.PNG)